torch/csrc/jit/codegen/cuda/lower_validation.cpp

```diff
@@ -247,6 +248,14 @@ void validateVectorize(Fusion* fusion) {
             "IterDomains to the left of the compute at point cannot be vectorized.");
         has_vectorize_dim = true;
       }
+
+      if (concrete_id->getParallelType() == ParallelType::MisalignedVectorize) {
+        TORCH_INTERNAL_ASSERT(
+            !tv->hasComputeAt() ||
+                tv->getComputeAtPosition() == tv->nDims() - 1,
+            "Only allow misaligned vectorization in the -2 computeAt position.");
```

Comment:
Where do we validate the vectorized axis is inner-most?

Reply:
`validateVectorizedTensors` in executor_utils and `validateVectorize` in lower_validation

Reply:
I'm trying to find where it is for `MisalignedVectorize`. I think `validateVectorizedTensors` only looks at `Vectorize`, which makes sense. Where should I look at in `validateVectorize` for `MisalignedVectorize`?

Reply:
It is in the same spot. 

https://github.com/csarofeen/pytorch/blob/dfc1690a7f0779a9111c7279e757414b6bee0d86/torch/csrc/jit/codegen/cuda/lower_validation.cpp#L119-L212

Reply:
Thanks! I found it's in the very end of that function.